x/multisig/keeper/keygen.go

```diff
@@ -79,6 +82,23 @@ func (k Keeper) DeleteKeygenSession(ctx sdk.Context, id exported.KeyID) {
 	k.getStore(ctx).Delete(getKeygenSessionKey(id))
 }
 
+// KeygenOptOut opts out the given participant of future keygens
+func (k Keeper) KeygenOptOut(ctx sdk.Context, participant sdk.AccAddress) {
```

Comment:
Can you add some events for both `KeygenOptOut` and `KeygenOptIn`?

Reply:
yes makes sense